src/engine/osaka.md

```diff
@@ -0,0 +1,106 @@
+# Engine API -- Osaka
+
+Engine API changes introduced in Osaka.
+
+This specification is based on and extends [Engine API - Prague](./prague.md) specification.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Structures](#structures)
+  - [BlobsBundleV2](#blobsbundlev2)
+  - [BlobAndProofV2](#blobandproofv2)
+- [Methods](#methods)
+  - [engine_getPayloadV5](#engine_getpayloadv5)
+    - [Request](#request)
+    - [Response](#response)
+    - [Specification](#specification)
+  - [engine_getBlobsV2](#engine_getblobsv2)
+    - [Request](#request-1)
+    - [Response](#response-1)
+    - [Specification](#specification-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Structures
+
+### BlobsBundleV2
+
+The fields are encoded as follows:
+
+- `commitments`: `Array of DATA` - Array of `KZGCommitment` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844), 48 bytes each (`DATA`).
+- `cell_proofs`: `Array of DATA` - Array of `KZGProof` (48 bytes each, type defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844), semantics defined in [EIP-7594](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7594.md)).
```

Comment:
this shouldn't be snake case imo, everything in the engine api uses camel case

Reply:
Will do along other potential changes later